index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title id="title">Elvis Presley</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body id="main">
+    <div id="img-div">
+      <img id="image" src="https://pymstatic.com/84696/conversions/frases-elvis-presley-wide.jpg" alt="">
+      <p id="img-caption" > “Values are like fingerprints. Nobody's are the same, but you leave 'em all over everything you do” </p>
+    </div>
+    <div id="tribute-info">
+      <h3> Fact Elvis Presley</h3>
+      <ul>
+        <li>Presley was born in Tupelo, Mississippi, 
+          and relocated to Memphis, Tennessee with his family when he was 13 years old.</li>
+        <li>In November 1956, Presley made his film debut in Love Me Tender. Drafted into military service in 1958, Presley relaunched his recording career 
+          two years later with some of his most commercially successful work.</li>
+        <li>With his rise from poverty to significant fame, Presley's success seemed to epitomize the American Dream. He is one of the best-selling music artists of all time, and was commercially successful in many genres, 
+          including pop, country, R&B, adult contemporary, and gospel.</li>
+        <li>Presley holds several records; the most RIAA certified gold and platinum albums, the most albums charted on the Billboard 200, and the most number-one albums by a solo artist on the 
+          UK Albums Chart and the most number-one singles by any act on the UK Singles Chart. </li>
+      </ul>
+      <a id="tribute-link" href="https://en.wikipedia.org/wiki/Elvis_Presley" target="_blank" rel="noopener noreferrer">More info about Elvis Presley</a>
+    </div>
```

Comment:
Instead of using multiple IDs for multiple elements, try using classes for multiple elements and only use IDs for specific elements that need special styling/functioning